sycl/include/CL/sycl/detail/pi.hpp

```diff
@@ -215,7 +215,13 @@ template <> inline void print<>(PiResult val) {
 }
 
 // cout does not resolve a nullptr.
-template <> inline void print<>(std::nullptr_t val) { print<void *>(val); }
+template <> inline void print<>(std::nullptr_t val) {
```

Comment:
```suggestion
template <> inline void print<>(std::nullptr_t) {
```
if you apply the proposed change below removing any use of `val`

Reply:
Oh!  that is a way around the compiler warning.  Thanks!

Reply:
done